src/molecules/account-menu.tsx

```diff
@@ -0,0 +1,38 @@
+import styles from './account-menu.module.css';
+
+import { Ref, forwardRef } from "react";
+
+export interface AccountMenuProps {
+  user: {
+    name: string;
+    initials: string;
+  } | null;
+  onLogout: () => void;
+  onLogin: () => void;
+}
+
+const AccountMenu = (
+  { user, onLogout, onLogin }: AccountMenuProps,
+  ref: Ref<HTMLDivElement>,
+) => {
+  if (isLoggedIn) {
+    return (
+      <div ref={ref} className={`${styles.accountMenu} surface-container on-surface-text`}>
+        <div className={styles.accountInfo}>
+          <span className={styles.userIcon}>{userInitial}</span>
+          <span className={styles.userName}>{userName}</span>
+        </div>
```

Comment:
```suggestion
  if (user) {
    return (
      <div ref={ref} className={`${styles.accountMenu} surface-container on-surface-text`}>
        <div className={styles.accountInfo}>
          <span className={styles.userIcon}>{user.initials}</span>
          <span className={styles.userName}>{user.name}</span>
        </div>
```
変更されたパラメータの使用箇所を修正しましょう